packages/builder/src/components/start/AppRow.svelte

```diff
@@ -18,14 +18,26 @@
   export let deleteApp
   export let unpublishApp
   export let releaseLock
+  export let editIcon
+
+  $: color =
+    $apps.find(filtered_app => app?.appId === filtered_app.appId)?.icon
+      ?.color || ""
```

Comment:
I'm guessing if `color` is an empty string like this then the inline style later down is ignored and it renders the icon with the defaullt spectrum color?

Reply:
yep exactly